<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoConfigStreamPage</class>
 <widget class="QWidget" name="AutoConfigStreamPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Basic.Settings.Stream.StreamType</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="streamType"/>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="preferHardware">
     <property name="toolTip">
      <string>Basic.AutoConfig.StreamPage.PreferHardwareEncoding.ToolTip</string>
     </property>
     <property name="text">
      <string>Basic.AutoConfig.StreamPage.PreferHardwareEncoding</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="doBandwidthTest">
     <property name="text">
      <string>Basic.AutoConfig.StreamPage.PerformBandwidthTest</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="region">
     <property name="title">
      <string>BandwidthTest.Region</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QCheckBox" name="regionAsia">
        <property name="text">
         <string>BandwidthTest.Region.Asia</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="regionUS">
        <property name="text">
         <string>BandwidthTest.Region.US</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="regionEU">
        <property name="text">
         <string>BandwidthTest.Region.EU</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="regionOther">
        <property name="text">
         <string>BandwidthTest.Region.Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="bitrate">
     <property name="suffix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>2500</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="bitrateLabel">
     <property name="text">
      <string>Basic.Settings.Output.VideoBitrate</string>
     </property>
     <property name="buddy">
      <cstring>bitrate</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>streamType</tabstop>
  <tabstop>preferHardware</tabstop>
  <tabstop>doBandwidthTest</tabstop>
  <tabstop>regionUS</tabstop>
  <tabstop>regionEU</tabstop>
  <tabstop>regionAsia</tabstop>
  <tabstop>regionOther</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
